internal/deployer/builder.go

```diff
@@ -0,0 +1,109 @@
+// Package deployer for project deployment
+package deployer
+
+import (
+	"fmt"
+	"net"
+	"strings"
+
+	"github.com/pkg/errors"
+	"github.com/threefoldtech/grid3-go/workloads"
+	"github.com/threefoldtech/grid_proxy_server/pkg/client"
+	"github.com/threefoldtech/grid_proxy_server/pkg/types"
+	"github.com/threefoldtech/zos/pkg/gridtypes"
+	"github.com/threefoldtech/zos/pkg/gridtypes/zos"
+)
+
+var (
+	vmFlist      = "https://hub.grid.tf/aelawady.3bot/abdulrahmanelawady-gridify-test-latest.flist"
+	vmCPU        = 2
+	vmMemory     = 2 * 1024
+	vmRootfsSize = 5 * 1024
+	vmEntrypoint = "/init.sh"
+	vmPublicIP   = true
+	vmPlanetary  = true
+)
+
+func constructNodeFilter() types.NodeFilter {
+	nodeStatus := "up"
+	freeMRU := uint64(2)
+	freeHRU := uint64(5)
+	freeIPs := uint64(1)
+	domain := true
+
+	filter := types.NodeFilter{
+		FarmIDs: []uint64{1},
```

Comment:
are we farm 1 on all networks? dev,qa,test,main? should we use FarmName instead?